scripts/release/prepare-release.sh

```diff
@@ -0,0 +1,370 @@
+        if [ "$current_branch" != "main" ]; then
+            log_info "Switching to main branch..."
+            git checkout main
+            git pull origin main
+        fi
+        
+        # Create and switch to a new branch
+        branch_name="release-prep-${new_version_v1#v}-${new_version_v2#v}"
+        git checkout -b "$branch_name"
+
+        # Commit updated CHANGELOG.md and UI submodule changes
+        git add CHANGELOG.md jaeger-ui
+        git commit -m "Prepare release ${new_version_v1} / ${new_version_v2}"
+
+        # Push branch
+        git push -u origin "$branch_name"
+
+        # Create the PR and robustly capture its URL
+        pr_out_file=$(mktemp)
+        if gh pr create \
+            --repo "$REPO" \
+            --title "$pr_title" \
+            --body "$pr_body" \
+            --base main \
+            --head "$branch_name" | tee "$pr_out_file" >/dev/null; then
+            pr_url=$(tr -d '\n' < "$pr_out_file")
+            log_success "Release PR created: $pr_url"
+            # Add changelog:skip label
+            gh pr edit "$pr_url" --add-label "changelog:skip"
+            log_success "Added changelog:skip label"
+        else
+            log_error "Failed to create PR"
+            rm -f "$pr_out_file"
+            rm -f "$temp_changelog"
+            exit 1
+        fi
+        rm -f "$pr_out_file"
+    fi
+    
+    # Clean up
+    rm -f "$temp_changelog"
+}
+
+# Create and push tags
+create_tags() {
+    log_info "Creating and pushing tags..."
+    
+    if $DRY_RUN; then
+        log_info "DRY RUN: Would execute the following commands:"
+        echo "git tag ${new_version_v1} -s"
+        echo "git tag ${new_version_v2} -s"
+        echo "git push upstream ${new_version_v1} ${new_version_v2}"
+        return
+    fi
+    
+    if $AUTO_TAG; then
+        log_info "Automatically creating and pushing tags..."
+        
+        # Create tags
+        git tag "${new_version_v1}" -s
+        git tag "${new_version_v2}" -s
+        
+        # Push tags
+        git push upstream "${new_version_v1}" "${new_version_v2}"
+        
+        log_success "Tags created and pushed successfully"
+    else
+        log_info "Manual tag creation mode. Please execute the following commands:"
+        echo
+        echo "git tag ${new_version_v1} -s"
+        echo "git tag ${new_version_v2} -s"
+        echo "git push upstream ${new_version_v1} ${new_version_v2}"
+        echo
+        read -p "Press Enter after you've created and pushed the tags..."
+    fi
+}
+
+# Main execution
+main() {
+    log_info "Starting automated release process..."
+    
+    check_prerequisites
+    get_current_versions
+    determine_next_versions
+    generate_changelog
+    create_release_pr
+    
+    log_info "Release PR creation completed!"
+    
+    if $AUTO_TAG; then
+        create_tags
+        log_success "Release automation completed successfully!"
+    else
+        log_info "To complete the release, please:"
```

Comment:
again this is repeating instructions that should be in the issue created by `start.sh` (i.e. the steps in the RELEASE.md)